### PR TITLE
AI-104: Fix delete contributions design issue

### PR DESCRIPTION
**Pull request by mukeshcompucorp**
**Before**

delete-issue

**After**

resolved-delete-issue

### CODE REVIEW

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_contributions.scss [hunk 1]
```diff
@@ -38,6 +38,51 @@
     }
   }
 
+  .CRM_Contribute_Form_Task_Delete {
+    box-shadow: $crm-contribute-form-shadow;
+    background: $crm-white;
+    margin-top: 15px;
+    border-radius: $border-radius-small;
```
Comment (meladawy) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_contributions.scss]: Please sort the properties

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_contributions.scss [hunk 2]
```diff
@@ -38,6 +38,51 @@
...
+
+    + p {
+      margin: 0;
+    }
+
+    .form-item {
+      background: $crm-white;
+      border-top: 1px solid $gray-lighter;
+      margin: 20px 20px 0;
+      padding: 20px 0 10px;
```
Comment (meladawy) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_contributions.scss]: plz sort properties

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_contributions.scss [hunk 3]
```diff
@@ -38,6 +38,51 @@
...
+      min-height: 43px;
+      border-radius: 0;
+    }
+
+    .crm-form-submit {
+      background: $brand-primary;
+      border-color: $brand-primary;
+      color: $crm-white;;
```
Comment (meladawy) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_contributions.scss]: Double semicolon

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_contributions.scss (hunk 1 above)
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_contributions.scss]: Fixed.

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_contributions.scss (hunk 2 above)
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_contributions.scss]: Fixed.

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_contributions.scss (hunk 3 above)
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_contributions.scss]: Fixed.

**Pull request merged**